researcher_ids/RI_Claims_V1.md

```diff
@@ -788,15 +793,15 @@ Types](#custom-passport-visa-types) and still be compliant.
 
 ### AcceptedTermsAndPolicies
 
--   The set of unique identifiers in the form of URLs to indicate that the
-    [Passport Visa Identity](#passport-visa-identity) or the
-    "[source](#source)" organization have acknowledged the specific terms,
-    policies, and conditions indicated by the URL.
+-   The [Passport Visa Identity](#passport-visa-identity) or the
+    "[source](#source)" organization has acknowledged the specific terms,
+    policies, and conditions (or meet particular criteria) as indicated by the
+    `value` field.
     
 -   The `value` field conforms to [URL Field](#url-fields) format.
 
--   The URLs SHOULD resolve to a public-facing, human readable web page that
-    describes the terms and policies.
+-   The URL SHOULD resolve to a public-facing, human readable web page that
+    describe the terms and policies.
 
 -   Example `value`: "<https://doi.org/10.1038/s41431-018-0219-y>"
     acknowledges the ethics terms as needed for
```

Comment:
"Bona fide researcher" is described using ResearcherStatus. This visa type is for expressing attestations. Suggesting: 
_Example value: "https://doi.org/10.1038/s41431-018-0219-y" acknowledges the attestations for for Registered Access._

Reply:
Changed your suggestion to "assertion" to remove attestations from the text.